test/integration/image_captioning/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
(cant comment directly on empty file `test/data/empty.txt`)
Do we need this empty.txt in the repository? If this is for tests only, it can be generated very easily in a temporary directory.

Reply:
About `2-conv-layer-*` files, I did not use temp files because those files were reused by 3 tests. So I do not need to create them every time. 

I will use temp files for all the ones you listed above.

Reply:
But each test should be able to run independently I think. These files are tiny (params file is 108k). Our synthetic digit data we create for our system tests is probably larger than this and its created in every test separately.

Reply:
Sure, I was already agree with you 😄

Reply:
ok :)